feature-group-definitions/font-face.yml

```diff
@@ -0,0 +1,8 @@
+spec: https://drafts.csswg.org/css-fonts-5/
```

Comment:
To me, font-face isn't really a feature. At least I've never referred to it as such. As a web dev, I want to load a custom font, use one that I've purchased, or use a freely available web font.

So, to me at least, the feature here is "Custom font" (there are probably better names for it).

Using the at-font-face rule is the way to make it work from CSS, but you can also use the (upcoming) Local Font Access API (https://chromestatus.com/feature/6234451761692672).

Also, the compat_features here should probably include the FontFace interface: https://developer.mozilla.org/en-US/docs/Web/API/FontFace.

Unless there's a parent group that's called "Custom fonts" that contains this group, and another one called local-fonts, and another called font-face-js-access, or something.

I guess I'm trying to understand what the right granularity here is.

Reply:
I think a common term for this feature is just "web fonts", which is also what https://caniuse.com/fontface has in the description.